lib/kitchen/verifier/inspec.rb

```diff
@@ -182,8 +182,9 @@ def resolve_config_inspec_tests
       # @return [Array<String>] array of suite directories or remote urls
       # @api private
       def collect_tests
-        # get local tests and get run list of profiles
-        (local_suite_files + resolve_config_inspec_tests).compact.uniq
+        # get run list of profiles or fall back to local tests
+        return resolve_config_inspec_tests.uniq if resolve_config_inspec_tests
```

Comment:
It looks like `resolve_config_inspec_tests` will _always_ return an array and the `if` condition will always be true, making the next line unreachable